scanpy/preprocessing/_simple.py

```diff
@@ -1112,7 +1125,7 @@ def _downsample_per_cell(X, counts_per_cell, random_state, replace):
 
 def _downsample_total_counts(X, total_counts, random_state, replace):
     total_counts = int(total_counts)
-    total = X.sum()
+    total = axis_sum(X)
```

Comment:
```suggestion
    total = X.sum()
```

Since this won't work with dask arrays, can we leave it untouched for now?